### PR TITLE
vendor update: helmclient

### PR DESCRIPTION
**Pull request by tomahawk28**
Toward https://github.com/giantswarm/giantswarm/issues/4953 
it would enable metrics from helmclient in chart-operator.

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -97,8 +97,8 @@ required = [
   version = "0.2.0"
 
 [[constraint]]
-  branch = "master"
   name = "github.com/prometheus/client_golang"
+  version = "0.9.3"
```
Comment (tomahawk28) [inline Gopkg.toml]: Syncing with most other giantswarm libraries.

**Comment by tomahawk28** [comment]
```
E0605 09:01:22.990129       1 portforward.go:391] an error occurred forwarding 38698 -> 5000: error forwarding port 5000 to pod f7c93c8289b4016f8a9edbe5002d8e4e7ca01efe6e9410a4223b2dc3b35394a3, uid : exit status 1: 2019/06/05 09:01:22 socat[12687] E connect(3, AF=2 127.0.0.1:5000, 16): Connection refused
2019/06/05 09:01:22 waiting for server at 6.842164996s: Get http://127.0.0.1:38698/cnr/api/v1/packages: EOF
```
Probably random error from circleCI?

**Comment by sslavic** [comment]
I've triggered a rerun of those failed tests.

**Comment by MarcelMue** [comment]
Please double check what is happening here and make sure e2e tests pass:
```
--- FAIL: TestChartValues (2.68s)
    chartvalues_test.go:38: could not get version bundle version GET https://api.github.com/repos/giantswarm/installations/contents/release/provider/aws.yaml: 401 Bad credentials []
```

**Comment by sslavic** [comment]
@tomahawk28 as @MarcelMue shared on slack, updating credentials/token env var did the trick.

**Comment by rossf7** [comment]
This is great but we also need to vendor in `draughtsman` and `cluster-operator`. Please test all 3 and then merge them together. From my experience `draughtsman` is the one that breaks most often.

We need to test tiller is upgraded in the `giantswarm` namespace in tenant clusters and `kube-system` for control plane.

For context. In TC we run tiller in giantswarm to allow customer to run tiller in kube-system. CP is our cluster so we just use kube-system.

**Comment by rossf7** [comment]
@tomahawk28 @sslavic Sorry the testing is quite complicated here. This is one example of the operational pain that tiller causes.

This is why medium term we will go tiller less. cc @puja108

**Comment by tomahawk28** [comment]
tested on kind cluster ```# HELP helmclient_library_event Histogram for events within the helmclient library.
# TYPE helmclient_library_event histogram
helmclient_library_event_bucket{event="ping_tiller",le="0.005"} 3
helmclient_library_event_bucket{event="ping_tiller",le="0.01"} 8
helmclient_library_event_bucket{event="ping_tiller",le="0.025"} 10
helmclient_library_event_bucket{event="ping_tiller",le="0.05"} 10
helmclient_library_event_bucket{event="ping_tiller",le="0.1"} 10
helmclient_library_event_bucket{event="ping_tiller",le="0.25"} 10
helmclient_library_event_bucket{event="ping_tiller",le="0.5"} 10
helmclient_library_event_bucket{event="ping_tiller",le="1"} 10
helmclient_library_event_bucket{event="ping_tiller",le="2.5"} 10
helmclient_library_event_bucket{event="ping_tiller",le="5"} 10
helmclient_library_event_bucket{event="ping_tiller",le="10"} 10
helmclient_library_event_bucket{event="ping_tiller",le="+Inf"} 10
helmclient_library_event_sum{event="ping_tiller"} 0.06530428099999999
helmclient_library_event_count{event="ping_tiller"} 10```

**Pull request merged**